### PR TITLE
add uploadthing preset

### PR DESCRIPTION
**Pull request by jln13x**

### CODE REVIEW

File: packages/core/src/presets.ts [hunk 1]
```diff
@@ -27,3 +27,14 @@ export const vercel = createEnv({
...
   runtimeEnv: process.env,
 });
+
+/**
+ * @see https://docs.uploadthing.com/getting-started/appdir#add-env-variables
+ */
+export const uploadthing = createEnv({
+  server: {
+    UPLOADTHING_SECRET: z.string(),
+    UPLOADTHING_APP_ID: z.string(),
```
Comment (juliusmarminge) [inline packages/core/src/presets.ts:37]: this variable isn't actually required so can be optional

Comment (jln13x) [reply packages/core/src/presets.ts]: done

Comment (juliusmarminge) [reply packages/core/src/presets.ts]: I meant just appId was optional. The secret is required 😅

Comment (jln13x) [reply packages/core/src/presets.ts]: there u go 👀

**Review by juliusmarminge — APPROVED** [review]

**Pull request merged**